app/src/androidTest/java/com/goldenraven/padawanwallet/intro/WalletRecoveryScreenTest.kt

```diff
@@ -0,0 +1,108 @@
+package com.goldenraven.padawanwallet.intro
+
+import android.util.Log
+import androidx.compose.ui.test.*
+import androidx.compose.ui.test.junit4.createAndroidComposeRule
+import com.goldenraven.padawanwallet.theme.PadawanTheme
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+
+class WalletRecoveryScreenTest {
+
+    @get:Rule
+    val composeTestRule = createAndroidComposeRule<IntroActivity>()
+
+    @Before
+    fun setUp() {
+
+        val onBuildWalletButtonClicked : (WalletCreateType) -> Unit = { walletCreateType ->
+            when (walletCreateType) {
+                is WalletCreateType.FROMSCRATCH -> Log.i("WalletRecoveryScreenTest", "Successful wallet init")
+                is WalletCreateType.RECOVER -> Log.i("WalletRecoveryScreenTest", "Successful wallet recovery")
+            }
+        }
+
+        composeTestRule.setContent {
+            PadawanTheme {
+                WalletRecoveryScreen(onBuildWalletButtonClicked = onBuildWalletButtonClicked)
+            }
+        }
+    }
+
+    // Backtick function names are introduced in API 30
```

Comment:
Yeah it's weird, from the comment (and answers online about this issue), you would think that it _should_ work for us (`compileSdk`, `buildToolsVersion`, `targetSdk` are all 30+), but clearly it's not. This somewhat contradicts [this Stack Overflow answer](https://stackoverflow.com/a/62597742/10523694), which says that it should work (even if it the IDE might underline it as an error). In my case it straight up doesn't compile, so yeah something is not quite right with that answer.  I even tried with the `@file:Suppress("IllegalIdentifier")` annotation mentioned in one of the comments, but of course that only fixes the IDE underline problem and still doesn't compile.

It does work if you bring the `mindSdk` variable to 30+, which I don't think we should do.

Anyway do you mind adding a line to the comment here just to make it clear why the test names are different? (the comment as is makes it sound like it should work for us even though it doesn't). Something like
```kotlin
// Backtick function names are introduced in API 30
// Would require setting minSdk = 30 in build.gradle.kts
```